op-deployer/pkg/deployer/bootstrap/flags.go

```diff
@@ -218,3 +226,10 @@ var Commands = []*cli.Command{
 		Action: ValidatorCLI,
 	},
 }
+
+func getDefaultOutfile(commandName, outfile, workdir string) string {
```

Comment:
Why do we need to provide a workdir? Have the user specify an infile as an arg to `verify`.

Reply:
Will delete